google_drive/public/js/google_calendar_edit.js

```diff
@@ -84,8 +84,12 @@ function GoogleCalendarEditBlock(runtime, element) {
         error_message_div.html();
         error_message_div.css('display', 'none');
         var handlerUrl = runtime.handlerUrl(element, 'studio_submit');
+
+        runtime.notify('save', {state: 'start', message: gettext("Saving")});
+
         $.post(handlerUrl, JSON.stringify(data)).done(function(response) {
             if (response.result === 'success') {
+                runtime.notify('save', {state: 'end'})
                 window.location.reload(false);
             } else {
                 error_message_div.html('Error: '+response.message);
```

Comment:
@rpenido I think it would be good to add the `notify("error")` here as well, like [this](https://github.com/open-craft/xblock-poll/blob/master/poll/public/js/poll_edit.js#L249)